image_writers_guide/sti.adoc

```diff
@@ -94,12 +76,15 @@ make all
 make install
 popd
 ----
+====
 
-=== run
+[[run]]
+*`run`*
 
 The `run` script is responsible for executing your application.
 
```

Comment:
[Suggestion]
s/run/Run Script

The `run` script executes your application.

[Question]
I've posted this question in another PR, but in the Terminology topic we state that the term "application' is no longer applicable in OpenShift v3. If this is the case then this reference to "application" is incorrect, or we need to update the Terminology topic.